src/styles/cylc/_footer.scss

```diff
@@ -3,3 +3,7 @@
   color: map-get($grey, 'darken-1');
   border-top: 1px solid #e7e7e7;
 }
+
+#core-footer {
+  z-index: 0;
```

Comment:
Out of interest why are we setting `z-index: 0`? Shouldn't have any effect?

Reply:
Hmmm, good question. Had a look with Firefox, and removing this value, the `#core-footer` element still has `z-index: 0` (I guess that's the default value).

Not sure why the guys from creativetim added it (they still have this in the `master` version - https://github.com/creativetimofficial/vuetify-material-dashboard/blob/2a3ea658e72e7109b7aa65be73abb6dfc547e3ea/src/components/core/Footer.vue#L45).

Any objection to removing it? We can add it back later if necessary, and it would be helping towards #295

Reply:
@kinow - do you want to just remove this line then (making the PR just a small deletion?)? Then we can merge it.

Reply:
Sure, done in a couple minutes :zap:

Reply:
Done